### ISSUE TITLE
Question: Data source

### ISSUE THREAD

**Issue by jonagoldman** [ernysans/laraworld#8]
I just want to know the source of the data in the json files in case I want to update with latest fixes (if there are any) or in general just to know. Thanks.

**Comment by ernysans** [comment]
@jonagoldman I'm sorry but I don't remember. I just know that at that time I had to retrieve the data from multiple sources. It has been some time sense the data was double checked. Please let me know if you find inconsistencies and if you can send me a PR and I will review it

**Comment by jonagoldman** [comment]
I noticed in this list of [UTC offsets](https://en.wikipedia.org/wiki/List_of_UTC_time_offsets) a few not available in the package, UTC+12:45, UTC+13:00, UTC+14:00, etc.. so I'am guessing some things might be outdated.

**Comment by carlost89** [comment]
Hi, Is there a way we can manually update the json files outside from vendors folder?

**Comment by ernysans** [comment]
@carlost89 Not really. Yon can send a PR If you find the information to be outdated

**Comment by jonagoldman** [comment]
I used https://timezonedb.com to retrieve [timezones](https://timezonedb.com/time-zones) and https://restcountries.eu to get countries and languages as a JSON. Then wrote a simple script to merge everything together and insert everything into the database.
